integration-tests/tests/run-notebook.js

```diff
@@ -48,11 +60,18 @@ const testRunNotebookFn = _.flow(
   await fillIn(frame, '//textarea', 'print(123456789099876543210990+9876543219)')
   await click(frame, clickable({ text: 'Run' }))
   await findText(frame, '123456789099886419754209')
-})
+}
+
 const testRunNotebook = {
   name: 'run-notebook',
   fn: testRunNotebookFn,
   timeout: 20 * 60 * 1000
 }
 
-module.exports = { testRunNotebook }
+const testRunNotebookWithV1Workspace = {
+  name: 'run-notebook-v1-workspace',
+  fn: testRunNotebookWithV1WorkspaceFn,
+  timeout: 20 * 60 * 1000
+}
+
+module.exports = { testRunNotebook, testRunNotebookWithV1Workspace }
```

Comment:
TOL: is it worth calling `testRunNotebook` `testRunNotebookWithV2Workspace` ?

Reply:
I am going to wait on that until CI takes a look! My thought is no, because it is whatever the default for creating a workspace is, not explicitly v2.

Reply:
I suggest leaving a comment to explain this.  I think the point here is that we expect `testRunNotebook` to run in a Workspace created via the latest mechanism (might be v1, v2 or some future version) and `testRunNotebookWithV1Workspace` is here while we transition to PPW, and after we've completed the PPW migration, we should no long need it.  I might also suggest creating a Jira issue in the PPW Cleanup epic to come back and clean-up/delete the tests no longer being used.